content/en/blog/_posts/2021-12-15-pod-security-admission-psa-beta.md

```diff
@@ -0,0 +1,620 @@
+---
+layout: blog
+title: 'Pod Security Admission (PSA) Graduates to Beta'
+date: 2021-12-15
+slug: pod-security-admission-psa-beta
+---
+
+**Authors:** Jim Angel (Google), Lachlan Evenson (Microsoft)
+
+_Editor's note: [Jim Angel](https://twitter.com/JimmAngel) and [Lachlan Evenson](https://twitter.com/LachlanEvenson) are both [CNCF Cloud Ambassadors](https://www.cncf.io/people/ambassadors/) and wear many hats in the open source Kubernetes community. Jim is a Cloud Consultant at Google and Lachlan is a Principal Program Manager on the Azure Core Upstream._
```

Comment:
>  Lachlan is a Principal Program Manager on the Azure Core Upstream.

either "Lachlan is a Principal Program Manager on the Azure Core Upstream team." or "Lachlan is a Principal Program Manager on Azure Core Upstream." would be better.

Reply:
I'd shift this paragraph to the foot of the article. We don't have a strong convention on this but I want readers to jump straight to the meat of the article.

Reply:
```suggestion
_Editor's note: [Jim Angel](https://twitter.com/JimmAngel) and [Lachlan Evenson](https://twitter.com/LachlanEvenson) are both [CNCF Cloud Ambassadors](https://www.cncf.io/people/ambassadors/) and wear many hats in the open source Kubernetes community. Jim is a Cloud Consultant at Google and Lachlan is a Principal Program Manager on the Azure Core Upstream team._
```

Reply:
Accidentally closed this one out.

The editors note blurb was used to establish credibility before continuing to read (as seen in similar blogs). I'd rather remove it entirely over moving it to the bottom.

Thoughts?

Reply:
It's the Kubernetes blog and we're discussing a feature graduation. _To me_ that provides all the gravitas we need. Omitting any note like this is a form of [countersignalling](https://en.wikipedia.org/wiki/Countersignaling).